sdcm/nemesis.py

```diff
@@ -529,19 +531,8 @@ def get_list_of_methods_compatible_with_backend(
             manager_operation: Optional[bool] = None,
             zero_node_changes: Optional[bool] = None,
     ) -> List[str]:
-        return self.get_list_of_methods_by_flags(
-            disruptive=disruptive,
-            supports_high_disk_utilization=supports_high_disk_utilization,
-            run_with_gemini=run_with_gemini,
-            networking=networking,
-            kubernetes=self._is_it_on_kubernetes() or None,
-            limited=limited,
-            topology_changes=topology_changes,
-            schema_changes=schema_changes,
-            config_changes=config_changes,
-            free_tier_set=free_tier_set,
-            manager_operation=manager_operation,
-            zero_node_changes=zero_node_changes
+        return self.nemesis_registry.get_list_of_methods_compatible_with_backend(
+            disruptive, supports_high_disk_utilization, run_with_gemini, networking, limited, topology_changes, schema_changes, config_changes, free_tier_set, manager_operation, zero_node_changes
```

Comment:
This approach is fragile, it is easy to mistakenly swap some of the values and we will get unexpected results.
Better to use key=value approach when we have such a big list of params.

Reply:
Fair point